packages/farcaster-js-neynar/src/v2/openapi/spec.yaml

```diff
@@ -1027,3 +1335,101 @@ paths:
           $ref: "#/components/responses/403Response"
         "500":
           $ref: "#/components/responses/500Response"
+  /farcaster/user/custody-address:
```

Comment:
We should probably add issues to track creating bindings for these new API endpoints

Reply:
agreed. Updates have been a bit more challenging than I would like them to be.

Reply:
Would you be up for making a quick github issue to track the new endpoints?